packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -19,7 +19,7 @@ import 'theme.dart';
 import 'tooltip.dart';
 
 /// Defines the layout and behavior of a [BottomNavigationBar].
-///
+/// For a sample on how to use these, please see [BottomNavigationBar].
```

Comment:
We usually leave a line between the leading sentence and any descriptions that follow.

We also usually leave a line between the See also section and the description.

```suggestion
///
/// For a sample on how to use these, please see [BottomNavigationBar].
///
```